junit5-jupiter-starter-bazel/junit5.bzl

```diff
@@ -65,10 +65,10 @@ def java_junit5_test(name, srcs, test_class = None, deps = [], runtime_deps = []
             kwargs.pop(arg)
 
     junit_console_args = []
-    if test_class:
-        junit_console_args += ["--select-class", test_class]
+    if test_package:
+        junit_console_args += ["--select-package", test_package]
     else:
-        fail("must specific 'test_class'")
+        fail("must specify 'test_package'")
```

Comment:
❓ Why do we have a default value for the parameter only to fail if it's not set?

Reply:
You're right. Does not make sense. I've changed it.